### ISSUE TITLE
Which advantage does the subproc environment have over the tmux environment?

### ISSUE THREAD

**Issue by kapsl** [SurrealAI/surreal#29]

**Comment by kaixindelele** [comment]
Wow, it’s secretly updated,I can't use Surreal-TMUX suddenly...

**Comment by LinxiFan** [comment]
We are sorry that the update isn't announced explicitly. We have deprecated tmux in favor of the new `subproc` mode. You need to run `pip install --upgrade symphony` first before updating the surreal repo. `subproc` should work out of the box on any environment.
Advantages:
1. Easy to debug, no more `surreal-tmux create` and then `surreal-tmux log` to check any error message. 
2. Easy to terminate, ctrl-C will kill all distributed processes. 
3. No more dependency on tmux. Different versions of tmux can behave slightly different, which once caused segfault for us. It was a big headache.

**Comment by kapsl** [comment]
Done that.
But now i got the problem that it doesn't fin the algorithm command
/bin/bash: surreal-ppo: command not found
I tried giving the path to ppo_configs.py directly in the algorithm argument, but then he tells that yaml could not be imported - probably because it is running in an anaconda environment?

**Comment by LinxiFan** [comment]
1. Have you activated virtual env? For surreal-subproc, you need to activate virtualenv yourself before launching.
2. Make sure you run `pip install -e Surreal/` again.

**Comment by kapsl** [comment]
Ok, I think a problem was that I ran it out of pycharm. (I still didn't get this working).
When using the conda environment in the bash it works.

**Comment by kaixindelele** [comment]
Hello, unfortunately, I also came across your problem. Obviously, the problem I mentioned above can't be solved at all. and did you solve it in pycharm?
....
best wish

**Comment by kaixindelele** [comment]
Creating new Experiment "<your_username>-test"
 --> Created process replay
/bin/bash: surreal-ddpg: command not found

I run my pycharm in the same path as I run it in terminal.

**Comment by kapsl** [comment]
I didn't try any further, starting it from command line also works well for me

**Comment by LinxiFan** [comment]
Yes, please launch SURREAL from command line. We can't predict PyCharm's behavior (might even be different across their versions).

**Comment by kaixindelele** [comment]
hi,LinxiFan:
   I really confused by the pycharm, yesterday night I debug surreal_subproc.py in pycharm，I also activated env before launch pycharm, and I changed this : parser.add_argument('experiment_name') to defualt like num_agents. then received nothing , I waited long time ,no errors,no info printed.
   so what's your version about pycharm ,or what's ide you debug this large project?
　Is it possible to debug and create experiments in IDE？
　really confused.smile cry manually
thank you a lot!
